### PR TITLE
Add missing tests for AnimationGroup 'remover' field #1846

### PR DESCRIPTION
**Pull request by Yelwo**
## Overview: What does this pull request change?

- Adding tests for this issue - ManimCommunity/manim/pull/1722
- PR for this issue - ManimCommunity/manim/issues/1846

## Motivation and Explanation: Why and how do your changes improve the library?

This PR adds graphic and unit tests for AnimationGroup field that was not previously being tested.

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Review by huguesdevimeux — APPROVED** [review]
SGTM

File: tests/test_composition.py [hunk 1]
```diff
@@ -97,3 +101,40 @@ def test_animationgroup_with_wait():
...
     assert timings == [(wait, 0.0, 1.0), (sqr_anim, 1.0, 2.0)]
+
+
+@pytest.mark.parametrize(
+    "animation_remover, animation_group_remover", [(False, True), (True, False)]
+)
+def test_animationgroup_is_passing_remover_to_animations(
+    animation_remover, animation_group_remover
+):
+    scene = Scene()
```
Comment (huguesdevimeux) [inline tests/test_composition.py:112]: This makes me realize that there should be a `Scene` factory, so people won't have to instantiate a scene in a test.

Comment (Yelwo) [reply tests/test_composition.py:112]: That's true, it would be also great to inject it to tests as fixture, as in graphic tests

Comment (huguesdevimeux) [reply tests/test_composition.py:112]: Yes, totally !

**Comment by huguesdevimeux** [comment]
Thanks for the contribution !

**Pull request merged**